### DISCUSSION TITLE
Use provided maven installation for dependnecy resolution

### DISCUSSION THREAD

**Discussion by Schalli1987** [eclipse-m2e/m2e-core#1803] [Q&A]
Actually it is not possible to change the maven version that is used for dependency resolution, also I've configured a custom maven installation in the properties.

Currently this leads to errors resolving openjfx artifacts, see e.g. [openjfx mailing list](https://mail.openjdk.org/pipermail/openjfx-dev/2024-May/047298.html) and [maven issue](https://issues.apache.org/jira/browse/MNG-8131).

So I  was wondering, if it is possible to use the selected maven installation also for dependency resolution instead using the embedded version?

**Comment by laeubi** [comment]
> So I was wondering, if it is possible to use the selected maven installation also for dependency resolution instead using the embedded version?

Because of technical reasons, this is currently not possible.
